itests/org.openhab.binding.mqtt.homeassistant.tests/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
```

Comment:
Are the pom files part of your PR intentionally?

Reply:
Yes and no.
Is there a reason, why the itests projects refere to `2.5.2-SNAPSHOT` in the `2.5.x`?
I thought this was a bug and updated it....

Reply:
All MQTT related tests seem to be 2.5.2. @Hilbrand Do you know this is intentionally?

Reply:
No not intentional. The mqtt tests in itests are actually disabled (see the itests pom.xml) because in the time we migrated to the new build system there were some problems with them so they were disabled if I remember correctly. Therefor these poms get missed when updating to a new version. To be correct they should point to the latest (2.5.9-SNAPSHOT) and maybe we need to look into getting these tests running again.